frontend/src/metabase/query_builder/components/SavedQuestionIntroModal.tsx

```diff
@@ -14,20 +17,23 @@ const getLabels = (question: Question) => {
 
   if (type === "question") {
     return {
+      image: EmptyQuestion,
       title: t`It's okay to play around with saved questions`,
-      message: t`You won't make any permanent changes to a saved question unless you click Save and choose to replace the original question.`,
+      message: t`You can save your edits as a new question, or choose to overwrite the original question.`,
     };
   }
 
   if (type === "model") {
     return {
-      title: t`It's okay to play around with models`,
-      message: t`You won't make any permanent changes to them unless you edit their query definition.`,
+      image: EmptyModel,
+      title: t`You can filter and summarize any model, and save your results as a question`,
```

Comment:
nit: Something feels a little unexpected to me about the line-heights in this modal. The subtext seems to have a larger line height than the title. Should this get a quick design review? Ignore this comment if the modal matches the Figma, of course.

Reply:
Replied directly from Linear - it didn't post it under your message, for some reason.
https://github.com/metabase/metabase/pull/54101#issuecomment-2674705565

Reply:
I love your attention to detail!